examples_utils/notebook_logging/README.md

```diff
@@ -1,29 +1,29 @@
-## GCLogger
+# GCLogger
```

Comment:
I removed use of the abbreviation "GCL" because there is already another Graphcore term (Graphcore Communication Library) that uses GCL.

Reply:
Thanks, yes I remember now!